### DISCUSSION TITLE
Aw, Snap! Error code: SIGILL in Chromium for Linux

### DISCUSSION THREAD

**Discussion by ellawk** [open-quantum-safe/oqs-demos#205] [Q&A]
Hi!
Great project. I have followed the instructions for building Chromium browser (for Linux) using the OQS-BoringSSL fork. In order to verify the browser and the TLS handshake, I connect to the test-server as explained in the instructions. As expected, I get a certificate warning, and want to add the test-certificate to the list of trusted certificates. But I cannot open the "Settings" page, I get the error "Aw, Snap! Something went wrong while displaying this webpage. Error code: SIGILL". 

In addition, when I start Chromium (by executing ./out/Default/chrome), I get the warning "API keys are missing. Some functionality will be disabled." I also notice some additional errors shown in the terminal.

Except for this, the browser seems to be working ok.

Any idea what's wrong? Help much appreciated, thanks in advance.

**Comment by baentsch** [comment]
Well, I'm afraid this is not entirely unexpected: We ceased to actively keep maintaining/updating oqs-chromium (and more importantly, the underlying  [oqs-boringss](https://github.com/open-quantum-safe/boringssl) library) as there has not been a great deal of interest for this. If you'd like to use a PQ-enabled browser, what about [oqs-epiphany](https://github.com/open-quantum-safe/oqs-demos/tree/main/epiphany)?

**Reply by ellawk** [reply]
Ah, I see. Thanks for the response, anyway!

**Reply by baentsch** [reply]
UW. Don't be shy stating interest -- also, a PR would be greatly welcomed to keep this alive.
